gen/src/namespace.rs

```diff
@@ -6,9 +6,10 @@ impl Api {
         match self {
             Api::CxxFunction(efn) | Api::RustFunction(efn) => &efn.name.namespace,
             Api::CxxType(ety) | Api::RustType(ety) => &ety.name.namespace,
+            Api::TypeAlias(ety) => &ety.name.namespace,
```

Comment:
Merge the branch with the above?

Reply:
Unfortunately, that's not possible, since those have different inner types.

Reply:
Acknowledged.